src/WCSimConstructPMT.cc

```diff
@@ -379,63 +379,25 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructPMT(G4String PMTName, G4Str
     ////////////////////
     /// 1-PMT support //
     ////////////////////
-    //54mm is position of full PMT support wrt inside of pressure vessel.
```

Comment:
Where did this all go?

Reply:
All the variables are changed according to the CAD drawing for all geometries.